keyboards/planck/keymaps/mwpeterson/keymap.c

```diff
@@ -0,0 +1,305 @@
+// from <https://github.com/noahfrederick/dots/tree/master/planck
+
+#include "planck.h"
```

Comment:
Could you replace this include with `#include QMK_KEYBOARD_H` instead? 

And could you remove the other includes (except for "version.h")?